### PR TITLE
Update local path provisioner version in the example folder

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:
Update local path prov to use version 0.0.6

### CODE REVIEW

File: examples/kind/test/e2e-kind.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ set -o pipefail
 readonly CT_VERSION=v2.2.0
 readonly KIND_VERSION=0.2.1
 readonly CLUSTER_NAME=chart-testing
-readonly K8S_VERSION=v1.13.4
+readonly K8S_VERSION=v1.14.1
```
Comment (jlegrone) [inline examples/kind/test/e2e-kind.sh]: Can we make the same change in https://github.com/helm/chart-testing/blob/d9dba63311e5d0524832bf35e9c83993a76af93d/e2e-kind.sh?

**Comment by cpanato** [comment]
now it is released: https://github.com/rancher/local-path-provisioner/releases

I will address the comments above as well

File: examples/kind/test/local-path-provisioner.yaml [hunk 2]
```diff
@@ -60,7 +60,7 @@ spec:
       serviceAccountName: local-path-provisioner-service-account
       containers:
       - name: local-path-provisioner
-        image: rancher/local-path-provisioner:v0.0.5
+        image: rancher/local-path-provisioner:v0.0.6
```
Comment (jlegrone) [inline examples/kind/test/local-path-provisioner.yaml]: Looks like there's another version out?

```suggestion
        image: rancher/local-path-provisioner:v0.0.7
```

Comment (scottrigby) [reply examples/kind/test/local-path-provisioner.yaml]: Now https://github.com/rancher/local-path-provisioner/releases/tag/v0.0.8. Moving pretty fast

Comment (cpanato) [reply examples/kind/test/local-path-provisioner.yaml]: 0.0.9 :) updated

File: examples/kind/test/e2e-kind.sh (hunk 1 above)
Comment (cpanato) [reply examples/kind/test/e2e-kind.sh]: done

Comment (cpanato) [reply examples/kind/test/e2e-kind.sh]: done

**Comment by cpanato** [comment]
@scottrigby @jlegrone @unguiculus PTAL

File: examples/kind/test/kind-config.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 kind: Cluster
-apiVersion: kind.sigs.k8s.io/v1alpha3
+apiVersion: kind.sigs.k8s.io/v1alpha2
```
Comment (unguiculus) [inline examples/kind/test/kind-config.yaml]: Why did you change this?

Comment (cpanato) [reply examples/kind/test/kind-config.yaml]: I will check because i did locally to use the new kind and the charts installation dod not work

Comment (unguiculus) [reply examples/kind/test/kind-config.yaml]: I have experienced an issue with kind 0.3.0. Might be due to this. We should skip 0.3.0 and go to 0.4.0 directly.

Comment (cpanato) [reply examples/kind/test/kind-config.yaml]: setting to 0.5.1
sorry for the long delay.... :(
